luminoth/models/fasterrcnn/rpn_target.py

```diff
@@ -90,10 +89,6 @@ def _build(self, all_anchors, gt_boxes, im_size):
         in the original Caffe implementation by Ross Girshick. Ideally we
```

Comment:
Remove this comment..

Reply:
Also, a simple explanation on what `_build` does on the Tensorflow level (subsample backgrounds and foregrounds)